README.md

```diff
@@ -1,14 +1,5 @@
+# Experiments and proposals for gRPC features.
 
-# The gRPC documentation and examples have been moved!
+The purpose of this repository is to have a single place for people to push code and various documentation that gravitate around experimenting with new features for gRPC. This repository isn't tested, and will be lightly curated, so all of the code here shouldn't be considered functional nor production-worty.
```

Comment:
Typo "worty", and the wording is a little awkward to my eye. Is

"so the code here should not be considered production-worthy (and it may not be functional at all)."

slightly better?